docs/README.md

```diff
@@ -3,101 +3,102 @@ sidebar_position: 1
-accessed via a [Git](https://git-scm.com/)-like API.
-* The [Zed language](language/README.md) is the system's pipeline language for performing
+* The [super data model](formats/zed.md) is the abstract definition of the data types and semantics
+that underlie the super-structured data formats.
+* The [super data formats](formats/README.md) are a family of
+[human-readable (Super JSON, SUP)](formats/zson.md),
+[sequential (Binary Super JSON, SUPZ)](formats/zng.md), and
+[columnar (Super Parquet, SPAR)](formats/vng.md) formats that all adhere to the
+same abstract super data model.
+* The [SuperPipe language](language/README.md) is the system's pipeline language for performing
 queries, searches, analytics, transformations, or any of the above combined together.
-* A  [Zed query](language/overview.md) is a Zed script that performs
+* A  [SuperPipe query](language/overview.md) is a script that performs
 search and/or analytics.
-* A [Zed shaper](language/shaping.md) is a Zed script that performs
+* A [SuperPipe shaper](language/shaping.md) is a script that performs
 data transformation to _shape_
-the input data into the desired set of organizing Zed data types called "shapes",
+the input data into the desired set of organizing super-structured data types called "shapes",
 which are traditionally called _schemas_ in relational systems but are
-much more flexible in the Zed system.
+much more flexible in SuperDB.
+* A [SuperDB data lake](commands/zed.md) is a collection of super-structured data stored
+across one or more [data pools](commands/zed.md#data-pools) with ACID commit semantics and
+accessed via a [Git](https://git-scm.com/)-like API.
 
 ## Digging Deeper
 
-The [Zed language documentation](language/README.md)
-is the best way to learn about `zq` in depth.
-All of its examples use `zq` commands run on the command line.
-Run `zq -h` for a list of command options and online help.
+The [SuperPipe language documentation](language/README.md)
+is the best way to learn about `super` in depth.
+All of its examples use `super` commands run on the command line.
+Run `super -h` for a list of command options and online help.
 
-The [Zed lake documentation](commands/zed.md)
-is the best way to learn about `zed`.
-All of its examples use `zed` commands run on the command line.
-Run `zed -h` or `-h` with any subcommand for a list of command options
-and online help.  The same language query that works for `zq` operating
-on local files or streams also works for `zed query` operating on a lake.
+The [`super db` documentation](commands/zed.md)
+is the best way to learn about the SuperDB data lake.
+All of its examples use `super db` commands run on the command line.
+Run `super db -h` or `-h` with any subcommand for a list of command options
+and online help.  The same language query that works for `super` operating
+on local files or streams also works for `super db query` operating on a lake.
 
 ## Design Philosophy
```

Comment:
This whole paragraph basically helps justify the lake and therefore should perhaps move or be dropped entirely, but I intentionally did not attempt that in this pass.